Tests/Shared/WaitUntil.swift

```diff
@@ -6,7 +6,7 @@ import Foundation
 import XCTest
 
 extension XCTestCase {
-    func waitUntil(timeout: TimeInterval = 0.1, _ action: (_ done: @escaping () -> Void) -> Void) {
+    func waitUntil(timeout: TimeInterval = 0.5, _ action: (_ done: @escaping () -> Void) -> Void) {
```

Comment:
Which test was failing because of this?

Reply:
I don't remember and I also can't find the action on Github that his failed :/